DataConnectors/GCP/ArmTemplate/GCPOnboarding.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "DataCollectionEndpoint_name": {
+      "defaultValue": "DataCollectionEndpoint",
+      "type": "String"
+    },
+    "Location": {
+      "defaultValue": "[resourceGroup().location]",
+      "type": "String"
+    },
+    "DataCollectionRule_name": {
```

Comment:
why the '_' is that a format you saw somewhere else? did it come for the export  functionally? check out other arm templates to see what the way is it is usually  done